project-base/storefront/components/Forms/Button/Button.tsx

```diff
@@ -40,14 +38,13 @@ export const Button: FC<Props> = forwardRef(
         return (
             <button
                 className={twMergeCustom(
-                    'inline-flex w-auto cursor-pointer items-center justify-center gap-2 text-center font-bold uppercase outline-none transition-all hover:no-underline',
-                    size === 'small' ? 'py-1 px-4 text-sm' : 'py-3 px-8 text-base',
```

Comment:
Ask Martin Valenta about this. The original button comes from a styleguide we followed. I know it's a pretty small change and we shouldn't care, but just to be sure. You can close this comment as you wish...

Reply:
Yup, don't worry. Everyone will be notified about this new design.